[2024-12-30T17:48:03.627+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.md_account_d scheduled__2024-12-29T00:00:00+00:00 [queued]>
[2024-12-30T17:48:03.637+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.md_account_d scheduled__2024-12-29T00:00:00+00:00 [queued]>
[2024-12-30T17:48:03.638+0000] {taskinstance.py:2170} INFO - Starting attempt 3 of 5
[2024-12-30T17:48:03.679+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): md_account_d> on 2024-12-29 00:00:00+00:00
[2024-12-30T17:48:03.695+0000] {standard_task_runner.py:60} INFO - Started process 450 to run task
[2024-12-30T17:48:03.700+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'md_account_d', 'scheduled__2024-12-29T00:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmp1r1ltebp']
[2024-12-30T17:48:03.703+0000] {standard_task_runner.py:88} INFO - Job 34: Subtask md_account_d
[2024-12-30T17:48:03.926+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.md_account_d scheduled__2024-12-29T00:00:00+00:00 [running]> on host 403e7aadd7ee
[2024-12-30T17:48:04.000+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='md_account_d' AIRFLOW_CTX_EXECUTION_DATE='2024-12-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-29T00:00:00+00:00'
[2024-12-30T17:48:04.016+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2024-12-30T17:48:04.061+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-30T17:48:04.074+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=md_account_d, execution_date=20241229T000000, start_date=20241230T174803, end_date=20241230T174804
[2024-12-30T17:48:04.115+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-30T17:48:04.308+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
